doc/source/tutorial-series-customize-the-client-pytorch.ipynb

```diff
@@ -294,7 +344,7 @@
    "source": [
     "This works as expected, two clients are training for three rounds of federated learning.\n",
     "\n",
-    "Let's dive a little bit deeper and discuss how Flower executes this simulation. Whenever a client is selected to do some work, `start_simulation` calls the function `numpyclient_fn` to create an instance of our `FlowerNumPyClient` (along with loading the model and the data).\n",
+    "Let's dive a little bit deeper and discuss how Flower executes this simulation. Whenever a client is selected to do some work, `run_simulation` launches the `ClientApp` object which in turn calls the function `numpyclient_fn` to create an instance of our `FlowerNumPyClient` (along with loading the model and the data).\n",
     "\n",
     "But here's the perhaps surprising part: Flower doesn't actually use the `FlowerNumPyClient` object directly. Instead, it wraps the object to makes it look like a subclass of `flwr.client.Client`, not `flwr.client.NumPyClient`. In fact, the Flower core framework doesn't know how to handle `NumPyClient`'s, it only knows how to handle `Client`'s. `NumPyClient` is just a convenience abstraction built on top of `Client`. \n",
```

Comment:
Should we add a couple of sentences to the new Flower Next here?

Reply:
Good point, I've added links to the guides on how to use `ClientApp` and the explainer doc for Flower Next.